scalafix-core/shared/src/main/scala/scalafix/config/CustomMessage.scala

```diff
@@ -0,0 +1,47 @@
+package scalafix
+package config
+
+import metaconfig.{Conf, ConfError, ConfDecoder, Configured, Metaconfig}
+import org.langmeta._
+import scalafix.internal.config.MetaconfigPendingUpstream.XtensionConfScalafix
+
+import scala.language.implicitConversions
+
+class CustomMessage[T](
```

Comment:
Now that this is in our public API, can we add 
```scala
type CustomMessage[T] = scalafix.config.CustomMessage[T]
val CustomMessage = scalafix.config.CustomMessage
```
in the scalafix package object? Rule authors should only need to `import scalafix._; import scala.meta._` to get started.